great-idea/css/index.css

```
@@ -63,4 +63,110 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+body {
+	width: 80%;
+	margin: auto;
+}
+
+header {
+	margin-top: 50px;
+}
+
+#topbar {
+	display: flex;
+	justify-content: space-between;
+}
+
+nav {
+	width: 800px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+
+nav a {
```

A small change, similar to the other comment is to add something subtle to all the links like an underline on hover.
```suggestion
nav a:hover {
text-decoration: underline;
```
Adding feedback when users interact with your page is a great way to make the user experience more enjoyable and less frustrating.